app/src/org/commcare/tasks/DataPullTask.java

```diff
@@ -377,6 +382,10 @@ private ResultAndError<PullTaskResult> handleSuccessResponseCode(
             updateUserSyncToken(syncToken);
 
             onSuccessfulSync();
+            if (trace != null) {
+                trace.putAttribute(CCPerfMonitoring.ATTR_SYNC_SUCESS, AnalyticsParamValue.SYNC_SUCCESS);

```

Comment:
we don't want to log failures ?

Reply:
@shubham1g5 the [description](https://docs.google.com/spreadsheets/d/1l5OiT5HCscpUW3h0xs9uPzxt9nHrfeaaArr0CuxxUVc/edit?gid=640482631#gid=640482631&range=C5) says `Duration of a full sync process with the server`, so my understanding is that this is to track complete sync durations. And I don't think there is any value in tracking the amount of time a failed sync takes.